addon/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "Test Pilot",
   "name": "testpilot-addon",
-  "version": "0.4.10",
+  "version": "0.4.10-dev-c1409ce",
```

Comment:
This seems weird. Was this on purpose, or some side effect of some deploy/build script?